Create docker-compose.yml

faddat opened the pull request:
........'cause why not, right?

Commit:
Create docker-compose.yml

........'cause why not, right?

Commit:
Update docker-compose.yml

array instead of singleton

MisterTea commented on docker-compose.yml:
```diff
@@ -0,0 +1,10 @@
+  build: .
+  ports:
+    - 80:3000
+  domainname: klouds.org
```
What is klouds.org?

Commit:
Update docker-compose.yml

klouds.org is my and @ozzadar's company.  Today we are a bit of a container consultancy, but in the future we will be something a bit more, specifically, we aim to organize a docker-hosting borganism (borganism because it assimilates compute into standardized containers,) and because resistance is futile.  We're using SDN + Containers on metal to deliver higher performance than public cloud at a lower cost.  We're also experimenting with micro-clusters that will do extremely well geographically dispersed.  Our customers should be able to host anything you can run in a container, <20ms from any place on earth.

So, that is Klouds.org.  Today, it's mostly An empty WWW / @, but that will change soon.
